drivers/rtaudio/audio_driver_rtaudio.cpp

```diff
@@ -117,65 +117,37 @@ Error AudioDriverRtAudio::init() {
 	mix_rate = GLOBAL_DEF("audio/mix_rate", 44100);
 
 	int latency = GLOBAL_DEF("audio/output_latency", 25);
-	// calculate desired buffer_size, taking the desired numberOfBuffers into account (latency depends on numberOfBuffers*buffer_size)
-	unsigned int buffer_size = closest_power_of_2(latency * mix_rate / 1000 / target_number_of_buffers);
```

Comment:
You can likely remove the assignment of `target_number_of_buffers` a few lines above.

Reply:
Yeah, I've just removed it.